composer.json

```diff
@@ -3,7 +3,7 @@
   "description": "Client library for the BSD Tools API",
```

Comment:
We should specify a "php" version in composer.json, as well as updating our PHPUnit from 3.X to 5.X (should be seamless, it's just 6 that makes a bunch of changes)